rollup.config.js

```diff
@@ -73,6 +76,7 @@ export default [{
 }, {
 	input:   'src/index.ts',
 	plugins: getPlugins(true),
+	external: ['@babel/runtime'],
```

Comment:
This is standalone bundle, here can't be any externals.

Reply:
Ah, ok. Removed.